services/movie.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from db.models import Movie, Genre, Actor
+
+from django.db.models import QuerySet
+
+from django.core.exceptions import ObjectDoesNotExist
+
+
+def get_movies(
+        genres_ids: list | None = None,
+        actors_ids: list | None = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__in=genres_ids)
+
+    if actors_ids:
+        movies = movies.filter(actors__in=actors_ids)
+
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    try:
+        movie = Movie.objects.get(id=movie_id)
+        return movie
+    except ObjectDoesNotExist:
```

Comment:
This is not so good pattern - if record doesn't exist the warning should happen, if you silently return None instead of a Movie it may lead to undepredictable program behaviour later